index.py

```diff
@@ -45,7 +51,9 @@
                 ),
             ],
             className="govuk-width-container",
+            **{"aria-live": "polite", "aria-atomic": "true"},
         ),
+        footer([dcc.Link("Accessibility statement", href="#")]),
```

Comment:
Has the href been changed to "#" because there's no accessibility page created by default? If so it might be worth adding a comment explaining that this should be remapped when the page is created.